.github/workflows/release.yml

```
@@ -2,10 +2,8 @@ name: Create OverlayPlugin Release
 
 on:
   push:
-    branches:
-      - main
-    paths:
-      - '**/AssemblyInfo.cs'
+    tags:
+    - '*'
```

This GitHub workflow's trigger was changed to `tags`, but there are no a workflow contribute a tag. Would it be better to add a new workflow yaml file to create a tag based on the version number changes?

By the way, in order to maintain the current yaml style, it would be better to add 2 spaces before an array item:
```suggestion
    tags:
      - '*'
```

---

Ah yes the old yaml file was creating a release build when AssemblyInfo.cs changed (which could also happen on non-version changes like copyright notice, but these probably don't change often).

I just changed it to tag since it made the most sense to me at the time... But given that the release workflow cancels itself if a tag already exists it could theoretically run on every push and only would create a new release in case the version of the main OverlayPlugin was incremented.

---

Actually looking at the entire release pipeline `validate_versions.py` would need to be adapted too, maybe I should have looked at https://github.com/OverlayPlugin/OverlayPlugin/pull/14 first lol

---

It looks like this is no longer happening on `tags` (thank you), but I wanted to add that the intention here was to make it so that releases could be done entirely by making a PR that bumped the version, without having to also then push a tag alongside it.